arcgis-ios-sdk-samples/Maps/Create and save a map/CreateSaveMapViewController.swift

```diff
@@ -52,8 +54,31 @@ class CreateSaveMapViewController: UIViewController, CreateOptionsViewController
         AGSAuthenticationManager.shared().oAuthConfigurations.add(config)
         AGSAuthenticationManager.shared().credentialCache.removeAllCredentials()
         
-        // Initially show the map creation UI.
-        performSegue(withIdentifier: "CreateNewSegue", sender: self)
+        // Temporarily unset the API key for this sample.
+        // Please see the additional information in the README.
+        let apiKey = AGSArcGISRuntimeEnvironment.apiKey
+        AGSArcGISRuntimeEnvironment.apiKey = ""
+        
+        let portal = AGSPortal(url: URL(string: "https://www.arcgis.com")!, loginRequired: true)
```

Comment:
```suggestion
        let portal = AGSPortal.arcGISOnline(withLoginRequired: true)
```
Not required, but would be better looking.